Gems/PhysX/Code/Source/PhysXCharacters/Components/CharacterControllerComponent.h

```diff
@@ -125,13 +125,16 @@ namespace PhysX
         float GetHalfForwardExtent() override;
         void SetHalfForwardExtent(float halfForwardExtent) override;
 
-        // carbonated begin enable_carbonated_1: Methods called from o3de-gruber
-#if defined(CARBONATED)
+#if defined(CARBONATED) // Methods added to setup character collider so that to differentiate character collisions from other collisions
```

Comment:
Minor - Move the comment to a different line

Reply:
Description comment moved below `#if defined(CARBONATED)` here and elsewhere